lib/io.rb

```diff
@@ -1,17 +1,21 @@
 class IO
-    def self.get_task_index
+    def self.get_task_id
         print "Which task? "
-        gets.chomp.to_i - 1
+        gets.chomp.to_i
     end
 
     def self.get
         gets.chomp
     end
 
     def self.print_task(task)
-        puts "Title: #{task.title}"
-        puts "Description: #{task.description}"
-        puts "Complete: #{task.is_complete}"
+        if task == nil
+            puts "This task does not exist."
```

Comment:
Should you ever get a `nil` task? Might be more appropriate to `raise` an error here.